flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -36,6 +36,8 @@ export EXIT_CODE=0
 
 echo "Flink dist directory: $FLINK_DIR"
 
+FLINK_VERSION=$(cat ${END_TO_END_DIR}/pom.xml | sed -n 's/[[:blank:]]\+<version>\([a-zA-Z0-9\.-]\+\)<\/version>/\1/p')
```

Comment:
The `sed` command does not work on MacOS. You use the command mentioned here:
https://unix.stackexchange.com/questions/98217/using-sed-to-extract-text-between-2-tags

Reply:
`sed` is used in all of our E2E tests.

Reply:
what exactly is the problem? Is `[[:blank:]` not supported?

Reply:
It seems that the `+` causes problems. I searched for it but all I could find was:
https://unix.stackexchange.com/questions/13711/differences-between-sed-on-mac-osx-and-other-standard-sed

Anyway, `sed -n 's/.*<version>\(.*\)<\/version>/\1/p'` runs fine.